scripts/release

```diff
@@ -45,45 +48,51 @@ main() {
   if [ ! -d "${reldir}/etcd" ]; then
     mkdir -p "${reldir}"
     cd "${reldir}"
-    git clone https://github.com/etcd-io/etcd.git --branch "${BRANCH}"
+    run git clone "${REPOSITORY}" --branch "${BRANCH}"
   fi
-  cd "${reldir}/etcd"
+  run cd "${reldir}/etcd" || exit 2
+  run git pull origin
+  run git reset --hard
```

Comment:
Do we really want to reset hard in a script? An alternative might be to check if the checkout is clean and error out if it is not.

Reply:
Done. PTAL